ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.13__periodics.yaml

```diff
@@ -115,13 +115,6 @@ tests:
       TEST_TYPE: main
       VERIFY_COMPLIANCE_OPERATOR_SCAN: "true"
     workflow: hypershift-ibmcloud-e2e
-- as: e2e-aws-ovn-mce-conformance
```

Comment:
Looks like you're not only enabling MCE 2.5 in aws, also cleaning some stuff, could you please specify that in the PR/Commit?